WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -75,11 +97,16 @@
 
 <script>
   $('#<%= @past_selected ? "past" : "present" %>').button('toggle');
+  $('#<%= @recent_selected ? "recent" : ""%>').button('toggle');
 
```

Comment:
If I understood correctly, this is meant to "press" the button when the page first loads. What I did is probably not the correct way, though.